### PR TITLE
Found date inconsistencies and fixed them.

### PR DESCRIPTION
**Pull request by bledsoef**
Found date inconsistencies and fixed them.

Fixes issues #415

### CODE REVIEW

File: app/templates/admin/adminLogs.html [hunk 1]
```diff
@@ -26,7 +26,9 @@ <h1 class="text-center mt-5 mb-3">Admin Logs</h1>
 <tbody>
   {% for entry in allLogs%}
   <tr>
-    <td>{{entry.createdOn}}</td>
+    {% set sortDate = entry.createdOn.strftime("%Y/%m/%d/%H/%M")%}
+    {% set displayDate = entry.createdOn.strftime('%m/%d/%Y %I:%M %p')%}
+    <td data-sort={{sortDate}}>{{displayDate}}</td>
```
Comment (BrianRamsay) [inline app/templates/admin/adminLogs.html]: Attribute values should have quotes around them

**Review by BrianRamsay — CHANGES_REQUESTED** [review]

File: app/templates/events/event_list.html [hunk 2]
```diff
@@ -130,8 +130,13 @@ <h2 class="accordion-header" id="accordion__header_{{program}}">
                     {% for event in events %}
                       <tr>
                         <td> <a href="/eventsList/{{event.id}}/edit" class="link-primary">{{event.name}}</a></td>
+<<<<<<< HEAD
+                        <td>{{event.startDate.strftime('%m/%d/%Y')}}</td>
+                        <td>{{event.timeStart.strftime('%I:%M %p')}}</td>
+=======
```
Comment (BrianRamsay) [inline app/templates/events/event_list.html]: don't commit merge conflict artifacts

**Review by BrianRamsay — CHANGES_REQUESTED** [review]
In addition to the review comment, Anderson has a suggestion for the time display on the event list.

**Comment by AndersonStettner** [comment]
For the time format: we do not want to show the 0 before the hour in 12 hour display and add spacing between the times and hyphen

File: app/templates/admin/adminLogs.html (hunk 1 above)
Comment (Boorclark) [reply app/templates/admin/adminLogs.html]: resolved

File: app/templates/eventView.html [hunk 3]
```diff
@@ -0,0 +1,95 @@
+{% set title = eventData.name %}
+{% extends "base.html"%}
+
+{% block styles %}
+    {{super()}}
+{% endblock %}
+
+{% block scripts %}
+    {{super()}}
+{% endblock %}
```
Comment (BrianRamsay) [inline app/templates/eventView.html]: If you aren't adding anything to these blocks, you don't need to have them in your template at all

File: app/templates/events/event_list.html [hunk 4]
```diff
@@ -67,7 +67,16 @@ <h1 class="text-center mt-5">Events List for {{selectedTerm.description}}</h1>
...
-            <td>{{event.timeStart.strftime('%I:%M %p')}}-{{event.timeEnd.strftime('%I:%M %p')}}</td>
+            <td>{{event.startDate.strftime('%m-%d-%Y')}}</td>
+            {% set startTime = event.timeStart.strftime('%I:%M %p') %}
+            {% set endTime = event.timeEnd.strftime('%I:%M %p') %}
+            {% if startTime[0] == '0' %}
+                {% set startTime = startTime[1:] %}
+            {% endif %}
+            {% if endTime[0] == '0' %}
+                {% set endTime = endTime[1:] %}
+            {% endif %}
```
Comment (BrianRamsay) [inline app/templates/events/event_list.html]: You can remove the leading zeroes in the format string, with `%-I` instead of `%I`

**Review by BrianRamsay — CHANGES_REQUESTED** [review]

File: app/templates/events/event_list.html (hunk 2 above)
Comment (Boorclark) [reply app/templates/events/event_list.html]: resolved

File: app/templates/eventView.html (hunk 3 above)
Comment (Boorclark) [reply app/templates/eventView.html]: resolved

File: app/templates/events/event_list.html (hunk 4 above)
Comment (Boorclark) [reply app/templates/events/event_list.html]: resolved

**Review by BrianRamsay — CHANGES_REQUESTED** [review]
You've got a conflict to resolve with development (event view page)

**Review by BrianRamsay — APPROVED** [review]

**Pull request merged**